heron/scheduler-core/src/java/com/twitter/heron/scheduler/UpdateTopologyManager.java

```diff
@@ -147,14 +145,10 @@ private void updateTopology(final PackingPlans.PackingPlan existingProtoPackingP
     TopologyAPI.Topology topology = getTopology(stateManager, topologyName);
     boolean initiallyRunning = topology.getState() == TopologyAPI.TopologyState.RUNNING;
 
-    // fetch the topology, which will need to be updated
-    TopologyAPI.Topology updatedTopology =
-        getUpdatedTopology(topologyName, proposedPackingPlan, stateManager);
-
     // deactivate and sleep
     if (initiallyRunning) {
       // Update the topology since the state should have changed from RUNNING to PAUSED
-      updatedTopology = deactivateTopology(stateManager, updatedTopology, proposedPackingPlan);
+      deactivateTopology(stateManager, topology, proposedPackingPlan);
```

Comment:
Good to have a comment here saying:
`Will throw exceptions internally if failed to xxxx`

Reply:
done